gems/sorbet/test/snapshot/run_one_bazel.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
+# Runs a single srb init test from gems/sorbet/test/snapshot/{partial,total}/*
+
+shopt -s dotglob
+
+# --- begin runfiles.bash initialization ---
+# Copy-pasted from Bazel's Bash runfiles library https://github.com/bazelbuild/bazel/blob/defd737761be2b154908646121de47c30434ed51/tools/bash/runfiles/runfiles.bash
+set -euo pipefail
+if [[ ! -d "${RUNFILES_DIR:-/dev/null}" && ! -f "${RUNFILES_MANIFEST_FILE:-/dev/null}" ]]; then
+  if [[ -f "$0.runfiles_manifest" ]]; then
+    export RUNFILES_MANIFEST_FILE="$0.runfiles_manifest"
+  elif [[ -f "$0.runfiles/MANIFEST" ]]; then
+    export RUNFILES_MANIFEST_FILE="$0.runfiles/MANIFEST"
+  elif [[ -f "$0.runfiles/bazel_tools/tools/bash/runfiles/runfiles.bash" ]]; then
+    export RUNFILES_DIR="$0.runfiles"
+  fi
+fi
+if [[ -f "${RUNFILES_DIR:-/dev/null}/bazel_tools/tools/bash/runfiles/runfiles.bash" ]]; then
+  # shellcheck disable=SC1090
+  source "${RUNFILES_DIR}/bazel_tools/tools/bash/runfiles/runfiles.bash"
+elif [[ -f "${RUNFILES_MANIFEST_FILE:-/dev/null}" ]]; then
+  # shellcheck disable=SC1090
+  source "$(grep -m1 "^bazel_tools/tools/bash/runfiles/runfiles.bash " \
+            "$RUNFILES_MANIFEST_FILE" | cut -d ' ' -f 2-)"
+else
+  echo >&2 "ERROR: cannot find @bazel_tools//tools/bash/runfiles:runfiles.bash"
+  exit 1
+fi
+# --- end runfiles.bash initialization ---
+
+repo_root="$PWD"
+
+# NOTE: using rlocation here because this script gets run from a genrule
+# shellcheck disable=SC1090
+source "$(rlocation com_stripe_ruby_typer/gems/sorbet/test/snapshot/logging.sh)"
+
+# ----- Option parsing -----
+
+# these positional arguments are supplied in snapshot.bzl
+ruby_package=$1
+output_archive="${repo_root}/$2"
+test_name=$3
+
+if [[ "${test_name}" =~ partial/* ]]; then
+  is_partial=1
+else
+  is_partial=
+fi
+
+info "├─ test_name:  ${test_name}"
+info "├─ output_archive:  ${output_archive}"
+info "├─ is_partial: ${is_partial:-0}"
+
+# ----- Environment setup -----
+
+# Add ruby to the path
+# shellcheck disable=SC1090
+PATH="${repo_root}/$(dirname "$(rlocation "${ruby_package}/ruby")"):$PATH"
+export PATH
+
+# Add bundler to the path
+# shellcheck disable=SC1090
+BUNDLER_LOC="${repo_root}/$(dirname "$(rlocation "gems/bundler/bundle")")"
+GEMS_LOC="$BUNDLER_LOC/../gems"
+PATH="$BUNDLER_LOC:$PATH"
+export PATH
```

Comment:
ooc, why are you always exporting on a newline instead of `export PATH="..."`?

Reply:
shellcheck prohibits this

Reply:
Yes, this is to avoid tripping that error when the shellcheck runs.

Reply:
I believe shellcheck only prohibits it when you're running a command on the inside. So for example

```bash
export BUNDLER_LOC="${repo_root}/$(dirname "$(rlocation gems/bundler/bundle)")"
```

is not ok because there's a nested command, which has its own return code separate from the `export` statement's return code. But for 

```bash
export PATH="$BUNDLER_LOC:$PATH"
```

which is just joining two strings, there's no nested command so it's fine.

I'm fine keeping it as it is—I just wanted to point out that I'm pretty sure this is not a shellcheck restriction.

Reply:
Ah, thanks for clearing that up!